scripts/setup_sibling_namespaces.py

```
@@ -0,0 +1,194 @@
+
+    sibling_agents = discover_sibling_agents(args.openclaw_json)
+    external_agents = [] if args.no_external else list(_KNOWN_EXTERNAL_AGENTS)
+    external_agents.extend(args.extra)
+    excluded = set(args.exclude) | {args.primary}
+    siblings = [s for s in (sibling_agents + external_agents) if s not in excluded]
+    # De-dupe while preserving order so the printed snippet is deterministic.
+    seen: set[str] = set()
+    siblings = [s for s in siblings if not (s in seen or seen.add(s))]
+
+    if not siblings:
+        print(f"No sibling or external agents found to bridge.")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)